src/internal/clusterstate/v2.1.0.go

```diff
@@ -5,18 +5,12 @@ import (
 
 	"github.com/pachyderm/pachyderm/v2/src/internal/migrations"
 	"github.com/pachyderm/pachyderm/v2/src/internal/storage/fileset"
-	enterpriseserver "github.com/pachyderm/pachyderm/v2/src/server/enterprise/server"
 )
 
 var state_2_1_0 migrations.State = state_2_0_0.
-	Apply("Move EnterpriseConfig from etcd -> postgres", func(ctx context.Context, env migrations.Env) error {
```

Comment:
this file is responsible for the migration from Pachyderm 2.0 to 2.1. We should avoid editing these functions. To remove the dependencies on the enterpriseserver package, I would just vendor the `enterpriseserver.EnterpriseConfigPostgresMigration` and `enterpriseserver.DeleteEnterpriseConfigFromEtcd` inline here so that we preserve the migration as it was, even though we are no longer using enterprise in 2.12